src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -141,17 +141,6 @@ const DateInputModalWrapper = styled.div<{
   border-radius: 8px;
 `
 
-const StyledDateInput = styled(DateInputForm)<{
```

Comment:
I'm removing this part since these styles are never being applied since `DateInputForm` [doesn't accept a className prop](https://emotion.sh/docs/styled#:~:text=styled%20can%20style%20any%20component%20as%20long%20as%20it%20accepts%20a%20className%20prop.)